synapse/api/constants.py

```diff
@@ -17,6 +17,9 @@
 
 """Contains constants from the specification."""
 
+# the max size of a (canonical-json-encoded) event
+MAX_PDU_SIZE = 65536
```

Comment:
Below the comment says the max size is 65535, not 65536, doesn't really matter but is a bit confusing. 😄

Reply:
fixed the comment, thanks.